# page 1110
std_msgs/Header header

BinaryHeader log_header

#Data
uint32 week
float64 seconds

string imu_status

int32 z_acceleration
int32 negative_y_acceleration
int32 x_acceleration

int32 z_gyroscope
int32 negative_y_gyroscope
int32 x_gyroscope